tests/cypress/e2e/all/header.cy.ts

```diff
@@ -92,4 +91,9 @@ describe('Header', () => {
       })
     })
   })
+
+  function waitIsLoaded() {
```

Comment:
Works for now but it's a bit weird and random to either select a proposal title or connect button to determine if the page is loaded. Perhaps there's a better way somehow?